### ISSUE TITLE
ERROR:  operator does not exist: public.hstore -> text

### ISSUE THREAD

**Issue by jpuris** [reorg/pg_repack#214]
PG version 9.4.24
pg_repack version 1.4.4

```sh
$ pg_repack --table=public.log --jobs=4 --dbname=some_db --port=6432 --echo
```
```
[..]
LOG: (query) INSERT INTO repack.table_24266068 SELECT logid,logtype_id,logasset_id,logdata,logcreated FROM ONLY public.log
LOG: (query) SELECT repack.disable_autovacuum('repack.table_24266068')
LOG: (query) COMMIT
LOG: Initial worker 0 to build index: CREATE INDEX index_25784033 ON repack.table_24266068 USING btree (logtype_id, ((logdata -> 'CompanyId'::text)), logcreated) TABLESPACE pg_default
LOG: Initial worker 1 to build index: CREATE INDEX index_25784032 ON repack.table_24266068 USING btree (((logdata -> 'token'::text))) TABLESPACE pg_default
LOG: Initial worker 2 to build index: CREATE INDEX index_25783897 ON repack.table_24266068 USING btree (logasset_id) TABLESPACE pg_default
LOG: Initial worker 3 to build index: CREATE UNIQUE INDEX index_25288642 ON repack.table_24266068 USING btree (logid) TABLESPACE pg_default
LOG: Command finished in worker 0: CREATE INDEX index_25784033 ON repack.table_24266068 USING btree (logtype_id, ((logdata -> 'CompanyId'::text)), logcreated) TABLESPACE pg_default
WARNING: Error with create index: ERROR:  operator does not exist: public.hstore -> text
LINE 1: ...table_24266068 USING btree (logtype_id, ((logdata -> 'Compan...
                                                             ^
HINT:  No operator matches the given name and argument type(s). You might need to add explicit type casts.
LOG: (query) ROLLBACK
LOG: (query) SELECT repack.repack_drop($1, $2)
LOG:     (param:0) = 24266068
LOG:     (param:1) = 4
```

**Problem**
It looks like pg_repack is struggling with `(logtype_id, ((logdata -> 'CompanyId'::text)), logcreated)` syntax
```
WARNING: Error with create index: ERROR:  operator does not exist: public.hstore -> text
```

It seems to be working just fine, if I do not include `--jobs` opt.
```bash
$ pg_repack --table=public.log --dbname=some_db --port=6432 --echo
```
```
[..]
LOG: (query) INSERT INTO repack.table_24266068 SELECT logid,logtype_id,logasset_id,logdata,logcreated FROM ONLY public.log
LOG: (query) SELECT repack.disable_autovacuum('repack.table_24266068')
LOG: (query) COMMIT
LOG: (query) CREATE INDEX index_25784033 ON repack.table_24266068 USING btree (logtype_id, ((logdata -> 'CompanyId'::text)), logcreated) TABLESPACE pg_default
LOG: (query) CREATE INDEX index_25784032 ON repack.table_24266068 USING btree (((logdata -> 'token'::text))) TABLESPACE pg_default
LOG: (query) CREATE INDEX index_25783897 ON repack.table_24266068 USING btree (logasset_id) TABLESPACE pg_default
LOG: (query) CREATE UNIQUE INDEX index_25288642 ON repack.table_24266068 USING btree (logid) TABLESPACE pg_default
LOG: (query) SELECT repack.repack_apply($1, $2, $3, $4, $5, $6)
[..]
```

Is this a known issue ? I do not mind doing this without `--jobs` opt, but it does slow the whole thing down quite a bit.

**Comment by Melkij** [comment]
Hello
It should already be fixed in #192. But unfortunately there was no new release...

**Comment by jpuris** [comment]
Ah great! Looked at it but was not sure it is exact same root cause. Thank you @Melkij!
